incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -13,6 +13,7 @@ This document describes common tasks you might want to accomplish using HNC.
   * [Select namespaces based on their hierarchies](#use-select)
   * [Delete a subnamespace](#use-subns-delete)
   * [Organize full namespaces into a hierarchy](#use-full)
+  * [Resolve conditions on a namespace](#resolve-cond)
```

Comment:
Oh sorry, one more thing - can you call this `#use-resolve-cond`? The "use-" prefix is for any non-admin tasks.

Reply:
done